spec/project_spec.rb

```diff
@@ -48,7 +48,7 @@
   let(:version) { Optimizely::VERSION }
   let(:impression_log_url) { 'https://logx.optimizely.com/v1/events' }
   let(:conversion_log_url) { 'https://logx.optimizely.com/v1/events' }
-  let(:project_instance) { Optimizely::Project.new(config_body_JSON, nil, spy_logger, error_handler, false, nil, nil, nil, nil, nil, [], {batch_size: 1}) }
+  let(:project_instance) { Optimizely::Project.new(datafile: config_body_JSON, logger: spy_logger, error_handler: error_handler, event_processor_options: {batch_size: 1}) }
```

Comment:
Do you want to change `error_handler: error_handler,` to `error_handler:,`?

Reply:
...and in other spots too?  Is the linter looking for this in the SDK like the testapp?

Reply:
We still support 3.0 and the shorthand syntax was introduced in 3.1